src/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -3612,13 +3612,13 @@ msgstr ""
 #: templates/pycontw-2020/_includes/ext/discord.html:123
 msgid ""
 "3.Join the Python Taiwan Discord server. <br>Python Taiwan Discord server "
-"invite link: <a href='https://discord.gg/YYAgUk7'>https://discord.gg/"
-"YYAgUk7</a> <br>In your Discord client App, click the button with plus sign, "
+"invite link: <a href='https://discord.gg/94hgCQv'>https://discord.gg/"
+"</a> <br>In your Discord client App, click the button with plus sign, "
```

Comment:
Is there anything missing? It looks like the link is not complete orz.

Reply:
ooops nice catch!